pkg/controller/velero/velero.go

```diff
@@ -38,6 +37,14 @@ func (r *ReconcileVelero) provisionVelero(reqLogger logr.Logger, namespace strin
 	locationConfig := make(map[string]string)
 	locationConfig["region"] = platformStatus.AWS.Region
 
+	// Use chinese mirror for the upstream image in AWS china regions
+	var veleroImage string
+	if locationConfig["region"] == "cn-north-1" || locationConfig["region"] == "cn-northwest-1" {
+		veleroImage = "gcr.azk8s.cn/heptio-images/velero:v1.1.0"
+	} else {
+		veleroImage = "gcr.io/heptio-images/velero:v1.1.0"
+	}
+
```

Comment:
Can we do something like this intead:
```golang
const (
  veleroImageRegistry = "gcr.io/heptio-images"
  veleroImageRegistryCN = "gcr.azk8s.cn/heptio-images"
  veleroImageTag = "velero:v1.1.0"
)
```
and then join them together in the function? That will ensure that it's clear how to move the tag in the future.

I'd also like to have `cnRegions` be a slice, that we can loop through to check if `locationConfig["region"]` is in them, which will allow us to easily add new regions in the future. This bit could also be extracted out into another function, so it can be unit tested.

If you need assistance or an example of how to do this, please let me know.